scripts/sql/normalize_sql.py

```
@@ -0,0 +1,379 @@
+      r'alter table\s+(\w+)\s+add constraint\s+\S+\s+foreign key\s*\(([^)]+)\)\s+references\s+(\w+)(?:\s*\(([^)]+)\))?',
+      stmt
+  )
+  if m:
+    t, cols, rt = m.group(1), m.group(2).strip(), m.group(3)
+    # [CHANGED] 참조 컬럼은 생략해 표준화(인라인 FK와 동일 포맷)
+    out_lines.append(f'fk {t} ( {cols} ) -> {rt}')
+    return True
+  return False
+
+
+def normalize(sql: str, strict_enums_env: str = None) -> str:
```

Ruff (0.12.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)